### PR TITLE
feat: added store for users

### PR DESCRIPTION
**Pull request by Vsevolod1020**

### CODE REVIEW

File: src/app/modules/profile/components/profile/profile.component.html [hunk 1]
```diff
@@ -10,9 +10,12 @@
       <li routerLinkActive="active">
         <a routerLink="./statistics">{{ "PROFILE.STATISTICS" | translate }}</a>
       </li>
-      <li routerLinkActive="active">
-        <a routerLink="./edit">{{ "PROFILE.EDIT" | translate }}</a>
+      <li routerLinkActive="active" *ngIf="getUserRole$ | async as userRole">
+        <a routerLink="./edit" *ngIf="userRole === 'couch' || userRole === 'administrator' ? true : false">{{ "PROFILE.EDIT" | translate }}</a>
```
Comment (nattallius) [inline src/app/modules/profile/components/profile/profile.component.html]: It's better to have prop in component like 
`ableToEditRoles = ['couch', 'administrator']`
and then in html check `*ngIf="ableToEditRoles.includes(userRole)"`

File: src/app/modules/profile/components/profile/profile.component.ts [hunk 2]
```diff
@@ -1,15 +1,31 @@
...
 
-  constructor() { }
+  isHR(role: any) {
+    role === 'HR' ? true : false;
+  }
+  isAdmin(role: any) {
+    role === 'administrator' ? true : false;
+  }
+  isCouch(role: any) {
+    role === 'couch' ? true : false;
```
Comment (nattallius) [inline src/app/modules/profile/components/profile/profile.component.ts]: You can simply write `return role === 'HR';`

**Review by nattallius — APPROVED** [review]

File: src/app/modules/profile/components/profile/profile.component.html (hunk 1 above)
Comment (Vsevolod1020) [reply src/app/modules/profile/components/profile/profile.component.html]: Ok, I've corrected

File: src/app/modules/profile/components/profile/profile.component.ts (hunk 2 above)
Comment (Vsevolod1020) [reply src/app/modules/profile/components/profile/profile.component.ts]: Yes, you are right)

**Review by Ars6300 — APPROVED** [review]

**Pull request merged**